notebooks/240-dolly-2-instruction-following/240-dolly-2-instruction-following.ipynb

```diff
@@ -18,6 +18,7 @@
     "\n",
```

Comment:
<p>I believe this part should be extended to switch between compressed and fp16 model like for device</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Is option for FP32 really necessary? I would suppose FP16 should be the only option besides INT8.

Reply:
I think we should keep only "fp16" and "8-bit compressed" versions.

Reply:
in the latest notebook version, we already use fp16 by default, so fp32 is out of notebook, please rebase